<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>571</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>391</width>
    <height>571</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>391</width>
    <height>571</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TicTacToe</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Logo.PNG</normaloff>Logo.PNG</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frm_TicTacToe">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>391</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(12, 160, 144);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btn_Restart">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>480</y>
       <width>371</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(21, 189, 172);
border-radius: 10px;
font-size: 30px;
color: rgb(238, 236, 211);</string>
     </property>
     <property name="text">
      <string>Restart Game</string>
     </property>
    </widget>
    <widget class="QFrame" name="frm_PlayArea">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>371</width>
       <height>371</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(12, 160, 144);
border-radius: 10px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="frm_NegOneOne">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>110</width>
        <height>110</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(21, 189, 172);
border-radius: 10px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="btn_TTT1">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>111</width>
         <height>111</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frm_ZeroOne">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>0</y>
        <width>110</width>
        <height>110</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(21, 189, 172);
border-radius: 10px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="btn_TTT2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>111</width>
         <height>111</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frm_OneOne">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>0</y>
        <width>110</width>
        <height>110</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(21, 189, 172);
border-radius: 10px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="btn_TTT3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>111</width>
         <height>111</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frm_OneZero">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>130</y>
        <width>110</width>
        <height>110</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(21, 189, 172);
border-radius: 10px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="btn_TTT6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>111</width>
         <height>111</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frm_ZeroZero">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>130</y>
        <width>110</width>
        <height>110</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(21, 189, 172);
border-radius: 10px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="btn_TTT5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>111</width>
         <height>111</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frm_NegOneZero">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>130</y>
        <width>110</width>
        <height>110</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(21, 189, 172);
border-radius: 10px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="btn_TTT4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>111</width>
         <height>111</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frm_NegOneNegOne">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>260</y>
        <width>110</width>
        <height>110</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(21, 189, 172);
border-radius: 10px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="btn_TTT7">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>111</width>
         <height>111</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frm_ZeroNegOne">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>260</y>
        <width>110</width>
        <height>110</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(21, 189, 172);
border-radius: 10px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="btn_TTT8">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>111</width>
         <height>111</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frm_OneNegOne">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>260</y>
        <width>110</width>
        <height>110</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(21, 189, 172);
border-radius: 10px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="btn_TTT9">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>111</width>
         <height>111</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QFrame" name="frm_ScoreBoard">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>371</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(12, 160, 144);
border-radius: 10px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="frm_PlayerScore">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(21, 189, 172)</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="lbl_AIScore">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>1</y>
         <width>181</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font-family: MS PGothic;
font-size: 40px;
color: rgb(238, 236, 211);</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frm_AIScore">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>20</y>
        <width>180</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(21, 189, 172)</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="txt_AIScore">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>1</y>
         <width>181</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font-family: MS PGothic;
font-size: 40px;
color: rgb(238, 236, 211);</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="lbl_AIDisplay">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>0</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(238, 236, 211);
background: ;
font-size: 16px;
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>AI</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_PlayerDisplay">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 16px;
font-weight: bold;
color: rgb(238, 236, 211);</string>
      </property>
      <property name="text">
       <string>Player</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
